development-defaults.ini

```diff
@@ -84,12 +98,29 @@ orange16 = "Orange 16"
 orange17 = "Orange 17"
 orange18 = "Orange 18"
 orange19 = "Orange 19"
-cabin01 = "Cabin 01 - GoldFinch - Music"
-cabin02 = "Cabin 02 - Ibis - Music"
-cabin03 = "Cabin 03 - Falcon - Music"
-cabin04 = "Cabin 04 - Blue jay (lot blue 9)"
-cabin05 = "Cabin 05 - Junco (Lot Yellow 25)"
-manor_house01 = "Manor House"
+orange20 = "Orange 20"
+orange21 = "Orange 21"
+orange22 = "Orange 22"
+orange23 = "Orange 23"
+
+dwelling01 = "Manor House"
+dwelling02 = "Bunk House"
+dwelling03 = “Blue Heron (Park Model) (unavailable)“
```

Comment:
These are smartquotes but you need regular double-quotes. The editor you're using is probably messing you up here.